src/index.js

```diff
@@ -1,93 +1,13 @@
-const fs = require("fs"),
-    http = require("http"),
-    path = require("path"),
-    methods = require("methods"),
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
+import express from 'express';
+import dotenv from 'dotenv';
 
-const isProduction = process.env.NODE_ENV === "production";
-
-// Create global app object
+dotenv.config();
 const app = express();
+app.get('/', (req, res) => res.status(200).send({
+  message: 'Welcome to Barefoot Nomad.'
+}));
 
-app.use(cors());
```

Comment:
Please restore this  and the following middleware because we need in this project